src/resolver.js

```diff
@@ -74,6 +73,31 @@ exports.resolve = (binaryBlob, path, callback) => {
       } else if (split[0] === 'Data') {
         cb(null, { value: node.data, remainderPath: '' })
       } else {
+        const values = {}
```

Comment:
Can you please add a comment saying that this is the case of a named link? It took me a while to figure that out.